config/global.ini.php

```diff
@@ -744,6 +744,13 @@
 
 [Tracker]
 
+; When enabled and a userId is set, then the visitorId will be automatically set based on the userId. This allows to
+; identify the same user as the same visitor across devices. It might also create a new visit as soon as a user logs in
+; or logs out. This won't be the case when this feature is disabled and it separates the userId from the visitorId.
+; Disabling this feature can be useful for example when using the third party cookie, and thus all Matomo sites use
```

Comment:
not really sure how to best explain when it would be useful. it's a quite specific case I guess

Reply:
I think its well explained :+1: